### ISSUE TITLE
[Accessibility][Jaws][Usage Report]: Screen reader reads "rows per page" twice—once normally and then again with its selected area.

### ISSUE THREAD

**Issue by olehnoskov-cengage** [cengage/react-magma#2006]
[A11Y-1110](https://jira.cengage.com/browse/A11Y-1110)

**Environment:**
Windows 11 / Chrome / Keyboard
Mac15/ Safari/ Voiceover

**Steps to Reproduce:**
1. Open the URL – https://react-magma.cengage.com/version/4.10.0/api/table/#pagination
2. Turn on a screen reader (JAWS/NVDA/VoiceOver).
3. Navigate to the text "Rows per page: and its selected area.
4. Observe that screen reader redundantly reads "rows per page" twice—first normally, then again with the selected option—causing repetitive and confusing announcements.

**Actual Result:**
The screen reader reads “rows per page” twice: once as the label, and immediately again reading the selected option and area, resulting in redundant information.

**Expected Result:**
The screen reader should announce “rows per page” once, followed by the selected value clearly, without unnecessary repetition.

**WCAG Guideline:**
1.3.1: Info and Relationships

**User Impact:**
Redundant reading can confuse users relying on screen readers, making navigation cumbersome and increasing cognitive load.

**Recommendation:**
Optimize ARIA labels and attributes to avoid repetitive announcements.

**Screencast:**
https://app.screencast.com/Tz1u16KgcmzWC

**Comment by olehnoskov-cengage** [comment]
This issue was investigated, and I didn't detect any issues with the current implementation:
The phrase `rows per page` is announced only once, when the user focuses on the select, and when the user updates the value of this select.

References, which work in the same way:
- https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/select
- https://mui.com/material-ui/react-select/

**Comment by olehnoskov-cengage** [comment]
We need to request activation keys for `Jaws` screen reader, because the  a11y vendor could reproduce it with Jaws.
The UI team confirmed that it's not reproducible for `VoiceOver` and `NVDA`.

<img width="912" height="426" alt="Image" src="https://github.com/user-attachments/assets/7881a166-6e6b-414b-9941-7d03599c98b5" />

**Comment by moathabuhamad-cengage** [comment]
can't be reduced with JAWS in Storybook Docs or CodeSandbox its announced once

https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/table--controlled-pagination
https://react-magma.cengage.com/version/4.10.0/api/table/#pagination
https://codesandbox.io/p/sandbox/xcst92?file=%2Findex.html

**Comment by nikitaorliak-cengage** [comment]
**Tested on:**
- V5: https://rc--upbeat-sinoussi-f675aa.netlify.app/api/table/#pagination
- V4: https://next--upbeat-sinoussi-f675aa.netlify.app/api/table/#pagination

Everything works as expected. No bugs found.
